### ISSUE TITLE
Looking for Pokemon.. every second?

### ISSUE THREAD

**Issue by ArmsAsuncion** [NECROBOTIO/NecroBot#3853]
Afaik, isn't scan delay should be >=7 (or 10) for the server to actually return valid data? Why does the bot scan (or look) for Pokemon every 1 second? 

Why not just scan every 10 seconds (or a preferred scan delay that can be set via config), while it's walking? So..

If next Pokestop is 100m away, and you are walking for 18km/h (5m/s), then it should only look for Pokemon every 50~ meters (in this case, only twice from the point you came from to the next Pokestop). This way, the bot won't set flags, if ever.

**Comment by nfettinger** [comment]
Update the bot to only look for new pokemon every 10 seconds? I believe this matches the new Niantic client.

**Comment by ArmsAsuncion** [comment]
Yes. It's 10 seconds right now. So if they could add this, it would be great.

**Comment by JohnnyVicious** [comment]
Seems to be still the case in 0.8.1, not to be pushy, just adding information :-)

[20:06:28] Looking for Pokemon...
[20:06:28] Looking for incense Pokemon...
[20:06:29] Looking for Pokemon...
[20:06:30] Looking for incense Pokemon...
[20:06:31] Looking for Pokemon...
[20:06:31] Looking for incense Pokemon...
[20:06:32] Looking for Pokemon...
[20:06:33] Looking for incense Pokemon...
[20:06:34] Looking for Pokemon...
[20:06:35] Looking for incense Pokemon...
[20:06:36] Looking for Pokemon...
[20:06:36] Looking for incense Pokemon...
[20:06:37] Looking for Pokemon...
[20:06:38] Looking for incense Pokemon...
[20:06:39] Looking for Pokemon...
[20:06:40] Looking for incense Pokemon...
[20:06:41] Looking for Pokemon...
[20:06:41] Looking for incense Pokemon...

**Comment by ArmsAsuncion** [comment]
With all these IP bans, I think this suggestion will help with the api calls limit, for like, a lot.

**Comment by JohnnyVicious** [comment]
Still the case in 0.8.5 and I get softbanned all the time alltho I'm not doing anything different compared to previous days, might be a good time to push this change.

[18:32:32] Looking for Pokemon...
[18:32:33] Looking for incense Pokemon...
[18:32:34] Looking for Pokemon...
[18:32:34] Looking for incense Pokemon...
[18:32:35] Looking for Pokemon...
[18:32:36] Looking for incense Pokemon...
[18:32:37] Looking for Pokemon...
[18:32:37] Looking for incense Pokemon...
[18:32:39] Looking for Pokemon...
[18:32:39] Looking for incense Pokemon...
[18:32:40] Looking for Pokemon...
[18:32:40] Looking for incense Pokemon...
[18:32:41] Looking for Pokemon...
[18:32:42] Looking for incense Pokemon...
[18:32:43] Looking for Pokemon...
[18:32:44] Looking for incense Pokemon...
